bzzeth/bzzeth_test.go

```diff
@@ -47,6 +63,40 @@ func newBzzEthTester() (*p2ptest.ProtocolTester, *BzzEth, func(), error) {
 	return protocolTester, b, teardown, nil
 }
 
+func newTestNetworkStore(t *testing.T) (prvkey *ecdsa.PrivateKey, netStore *storage.NetStore, cleanup func()) {
+	prvkey, err := crypto.GenerateKey()
+	if err != nil {
+		t.Fatalf("Could not generate key")
+	}
+
+	dir, err := ioutil.TempDir("", "localstore-")
+	if err != nil {
+		t.Fatalf("Could not create localStore temp dir")
+	}
+
+	bzzAddr := network.PrivateKeyToBzzKey(prvkey)
+	localStore, err := localstore.New(dir, bzzAddr, nil)
+	if err != nil {
+		os.RemoveAll(dir)
+		t.Fatalf("Could not create localStore")
+	}
+
+	netStore = storage.NewNetStore(localStore, bzzAddr, enode.ID{})
```

Comment:
please pass the enode id to netstore

Reply:
This is where we create it.. in all other places we will take this enode id from netstore.